README.md

```diff
@@ -25,18 +25,16 @@ This will ensure that the `django-admin` command is available in your shell. Fro
 you can create a new project with the following command:
 
 ```shell
-django-admin startproject --template path/to/django-template/template/ --extension py,env,sh,toml,yml --exclude .venv <project_name>
+django-admin startproject --template path/to/django-template/template/ --extension py,env,sh,toml,yml --exclude nothing <project_name>
 ```
 
 > [!NOTE]
-> When starting a new Django project using a custom template, some directories and files
-> that begin with a dot (.) are not automatically included. This exclusion typically affects
-> important folders like `.github`, which is commonly used for GitHub Actions CI/CD workflows.
-> However, from Django 4.0, you can ensure that directories like `.github` are properly
-> copied to your new project. You can achieve this by using the `--exclude` option during
-> project initialization. If you omit this option, directories beginning with a dot, including
-> `.github/workflows`, will be left out.
-
+>When initiating a Django project with a custom template, be aware that directories starting with
+>a dot (e.g., `.github` for GitHub Actions workflows) are not included by default. A workaround
+>from Django 4.0 onwards involves using the `--exclude` option with the `startproject` command.
+>Oddly, specifying `--exclude` with a non-existent directory can allow these dot-prefixed
+>directories to be copied. This trick can ensure that essential configurations like `.github` are
+>included in your project setup.
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought (non-blocking):** ‏I think we are rapidly approaching the point at which we need our own wrapper script for the generating a new project. This would allow us to do extra things like copy across the `.github` folder without resorting quirks, and also automatically add variants when we get some.

Not something we need right now though.

Reply:
I agree Jon 👍🏻